framework/web/JsonResponseFormatter.php

```diff
@@ -96,12 +124,20 @@ protected function formatJson($response)
      */
     protected function formatJsonp($response)
     {
-        $response->getHeaders()->set('Content-Type', 'application/javascript; charset=UTF-8');
-        if (is_array($response->data) && isset($response->data['data'], $response->data['callback'])) {
-            $response->content = sprintf('%s(%s);', $response->data['callback'], Json::htmlEncode($response->data['data']));
+        if (is_array($response->data)
```

Comment:
Is this code style refactoring a subject of this patch?

Reply:
You mean formatting change? It's fine.

Reply:
I mean that the subject of this patch is to add the content type property not to refactor stuff. Never mind.